packages/mobile/src/invite/saga.ts

```diff
@@ -213,7 +213,7 @@ export function* redeemInviteSaga(action: RedeemInviteAction) {
         throw e
       }
     }
-    Logger.debug(TAG + '@redeemInviteCode', 'Added temp account to wallet: ' + tempAccount)
```

Comment:
I've found this string useful in the past.

Reply:
Okay, will restore for now